samples/crypto/kmu_usage_nrf54l/README.rst

```diff
@@ -0,0 +1,142 @@
+.. _crypto_kmu_usage:
+
+Crypto: KMU usage (nRF54L series)
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The KMU usage sample shows how to generate and use various persistent keys using the Platform Security Architecture (PSA) APIs.
+Persistent keys are stored in the Key Management Unit (KMU) of the device and retain their value between resets.
+The implementation of the PSA ITS API is provided by the Oberon PSA Crypto, which implements the driver wrapper role
+and determines which driver to use for each cryptographic operation.
+For this sample CRACEN driver is used.
+
+The sample is also used to demonstrate the usage of the pre-provisioned keys (`via nRF Util <https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/nrf54l/kmu_provision.html#provisioning_keys_to_the_board>`_).
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+Overview
+********
+
+In this sample, four different keys/keypairs are used:
+
+1. AES-128;
+#. ECDSA key pair;
+#. EdDSA key pair.
+
+The sample performs the following operations:
+
+1. Initialization of the Platform Security Architecture (PSA) API.
+#. Generation of a persistent AES 128-bit key and ECDSA key pair.
+#. Removal of the keys from RAM.
+#. Using both pre-provisioned and generated keys/key pairs to encrypt/descrypt
+   or sign/verify a message or its hash.
+#. Cleanup.
+   All persistent keys generated by this sample will be removed from the KMU, while
+   all pre-provisioned keys remain in KMU between resets.
+
+.. note::
+   The read-only type of persistent keys cannot be destroyed with the ``psa_destroy_key`` function.
+   The ``PSA_KEY_PERSISTENCE_READ_ONLY`` macro is used for read-only keys.
+   The key ID of a read-only key is writable again after a full erase of the device memory.
+   Use the ``west -v flash --erase`` command for the full erase.
+
+.. note::
+   Builds without TF-M and all nRF54L15 builds use the :ref:`hardware unique key (HUK) <lib_hw_unique_key>` to encrypt the key before storing it.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/crypto/kmu_usage_nrf54l`
+
+.. include:: /includes/build_and_run_ns.txt
+
+Keys generation and provisioning
+==================================================
+
+Prior to program the device the following keys/key pairs must be generated:
+
+1. ed25519:
+   Refer to the `Managing keys <https://docs.nordicsemi.com/bundle/ncs-latest/page/mcuboot/imgtool.html#managing_keys>`_ section of the MCUBoot documentation.
```

Comment:
Add a link target to the /doc/nrf/links.txt file and use it as advised in https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/dev_model_and_contributions/documentation/styleguide.html#linking